### PR TITLE
Operator init and remove commands

### PR DESCRIPTION
**Pull request by ostromart**
This manages installing and removing the operator controller.

### CODE REVIEW

File: data/operator/templates/clusterrole_binding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: istio-operator
+subjects:
+- kind: ServiceAccount
+  name: istio-operator
+  namespace: istio-operator
```
Comment (morvencao) [inline data/operator/templates/clusterrole_binding.yaml]: change namespace to `{{.Values.operatorNamespace}}`?

File: cmd/mesh/operator-init.go [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+}
+
+// operatorInit installs the Istio operator controller into the cluster.
+func operatorInit(args *rootArgs, oiArgs *operatorInitArgs, l *logger, apply manifestApplier) {
+	initLogsOrExit(args)
+
+	// Error here likely indicates Deployment is missing. If some other K8s error, we will hit it again later.
+	already, _ := isControllerInstalled(oiArgs.kubeConfigPath, oiArgs.context, oiArgs.operatorNamespace)
+	if already {
+		l.logAndPrintf("Operator controller is already installed in %s namespace, updating.", oiArgs.istioNamespace)
```
[the remark this answers is not in the document]
Comment (ostromart) [reply cmd/mesh/operator-init.go]: Done.

File: cmd/mesh/operator-init.go [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+		return "", "", fmt.Errorf("Could not read values from file %s: %s", filePath, err)
+	}
+	customResource = string(b)
+	istioNamespace = mergedICPS.DefaultNamespace
+	return
+}
+
+// chartsRootDir, helmBaseDir, componentName, namespace string) (TemplateRenderer, error) {
+func renderOperatorManifest(args *rootArgs, oiArgs *operatorInitArgs, l *logger) (string, error) {
+	r, err := helm.NewHelmRenderer("", "../operator", "Operator", oiArgs.operatorNamespace)
```
[the remark this answers is not in the document]
Comment (ostromart) [reply cmd/mesh/operator-init.go]: Done.

File: data/operator/templates/clusterrole_binding.yaml (hunk 1 above)
Comment (ostromart) [reply data/operator/templates/clusterrole_binding.yaml]: Done.

**Review by morvencao — APPROVED** [review]

**Pull request merged**